docs-source/content/userguide/istio/istio.md

```diff
@@ -0,0 +1,275 @@
+other components and services that have tracing enabled.
+
+You can learn more about Istio at [Istio](https://istio.io/latest/docs/concepts/what-is-istio/)
+
+#### Limitations
+
+The current support for Istio has these limitations:
+
+* It is tested with Istio 1.4.2 and later (up to 1.5.4), however it is tested with both single and
+  multicluster installations of Istio.
+* You cannot expose any of the default channels, any attempt will result in an error when deploying the domain.  
+* In order to use WLST command, you can define a network access point (NAP) in your WebLogic Domain and expose it as a `NodePort` 
+in your domain resource YAML instead of accessing the channel through the Istio mesh network.  
+
+#### Using the operator with Istio support
+
+{{% notice note %}}
+These instructions assume that you are using a Kubernetes cluster with
+[Istio Installation](https://istio.io/latest/docs/setup/install/) installed and configured already.  The operator will not install
+Istio for you.
+{{% /notice %}}
+
+You can deploy the operator into a namespace which has Istio automatic sidecar
+injection enabled.  Before installing the operator, create the namespace you
+wish to run the operator in, and label it for automatic injection.
+
+```
+$ kubectl create namespace weblogic-operator
+$ kubectl label namespace weblogic-operator istio-injection=enabled
+```
+
+After the namespace is labeled, you can install the operator using the normal
+method.  When the operator pod starts, you will notice that Istio automatically
+injects an `initContainer` called `istio-init` and the envoy container `istio-proxy`.
+
+You can check this using the following commands:
+
+```
+$ kubectl --namespace weblogic-operator get pods
+$ kubectl --namespace weblogic-operator get pod weblogic-operator-xxx-xxx -o yaml
+```
+
+In the second command, change `weblogic-operator-xxx-xxx` to the name of your pod.
+
+#### Creating a domain with Istio support
+
+You can configure your domains to run with Istio automatic sidecar injection enabled.
+Before creating your domain, create the namespace you wish to run the domain in,
+and label it for automatic injection.
+
+```
+$ kubectl create namespace domain1
+$ kubectl label namespace domain1 istio-injection=enabled
+```
+
+To enable the Istio support for a domain, you need to add the
+`configuration` section to your domain custom resource YAML file as shown in the
+following example:  
+
+```
+apiVersion: "weblogic.oracle/v7"
+kind: Domain
+metadata:
+  name: domain2
+  namespace: domain1
+  labels:
+    weblogic.resourceVersion: domain-v2
+    weblogic.domainUID: domain2
+spec:
+  ... other content ...
+  configuration:
+    istio:
+      enabled: true
+      readinessPort: 8888
+```
+
+To enable the Istio support, you must include the `istio` section
+and you must set `enabled: true` as shown.  The `readniessPort` is optional
+and defaults to `8888` if not provided, it is used for readiness health check.
```

Comment:
provided, it is used for readiness health check. -> provided; it is used for readiness health check. (semi-colon joins two complete sentences)